subprojects/core/src/main/java/org/gradle/api/artifacts/ResolvableDependencies.java

```diff
@@ -60,12 +61,12 @@
     FileCollection getFiles();
 
     /**
-     * Returns a view of this set containing files with the given format.
+     * Returns a view of this set containing files matching the requested attributes.
      *
      * @since 3.3
      */
     @Incubating
-    FileCollection getFiles(String format);
+    FileCollection getFiles(Map<?, ?> attributes);
```

Comment:
Alright, if we want to have this functionality as well (matching against more attributes than the one in the configuration), we need the `attributes` parameter everywhere.